recipes/flask-marshmallow/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "flask-marshmallow" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "83e2a3bb767a97db63c23a84345430cd3fda51615e7e99131a6b313295f6b7f0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
+    - marshmallow >=1.2.0
+    - six >=1.9.0
```

Comment:
Please drop anything from `build` that is not strictly a requirement at build time. Probably just need `python` and `setuptools`.